components/03-organisms/menu/primary-nav/_yds-primary-nav.scss

```diff
@@ -8,6 +8,8 @@
   --color-navigation-expanded-item: var(--color-basic-brown-gray);
 }
 
+$menu-sub-max-width: 300px;
```

Comment:
@ModulesUnraveled Do you have a project example of where you've checked for viewport edge? I'll do some internet searching but would be interested if you have example code you can point me to.

Reply:
Centered menu variation has been fixed. Because a max-width is applied to the drop-downs, I wasn't able to see an issue with drop-downs which appear second to last, for example: 

![main-menu-alignment-fixes-right-side2](https://user-images.githubusercontent.com/366413/202577288-8a3ea84c-f14e-4fef-a4a2-50efdd67db6a.gif)

Wondering if we need JS detection?